crates/pallets/starknet/src/lib.rs

```diff
@@ -325,6 +324,24 @@ pub mod pallet {
     #[pallet::getter(fn l1_messages)]
     pub(super) type L1Messages<T: Config> = StorageValue<_, BTreeSet<Nonce>, ValueQuery>;
 
+    /// ChainID for the palle'a, 'a, t startknet
+    #[pallet::storage]
+    #[pallet::getter(fn chain_id)]
+    pub type ChainIdStorage<T> = StorageValue<_, Felt252Wrapper, ValueQuery, DefaultChainId>;
+
+    /// Default ChainId SN_GOERLI_TESNET id
+    pub struct DefaultChainId {}
+
+    impl Get<Felt252Wrapper> for DefaultChainId {
+        fn get() -> Felt252Wrapper {
+            Felt252Wrapper(starknet_ff::FieldElement::from_mont([
+                3753493103916128178,
+                18446744073709548950,
+                18446744073709551615,
+                398700013197595345,
+            ]))
```

Comment:
This is defined as a const in `crates/primitives/chain-id/src/lib.rs`.
I would rather use it from there